funnel/templates/scan_badge.html.jinja2

```diff
@@ -72,7 +72,7 @@
     $(function() {
 
       var eventConfig = {
-        checkinApiUrl: "{{ url_for('checkin_puk', profile=project.profile.name, project=project.name, name=event.name, puk='puk') }}",
+        checkinApiUrl: "{{ url_for('EventParticipantCheckinView_checkin_puk', profile=project.profile.name, project=project.name, event=event.name, puk='puk') }}",
```

Comment:
This should be `event.url_for(...)`

Reply:
This endpoint is a bit weirdly handled on the front end. Moving it to `event.url_for()` will require redoing front end as well. I'll check with vidya if it's possible. I thought we'll come and revisit this `puk` issue later and for now do this.